public/src/bc5SupportDev/java/com/revenuecat/purchases/models/BC5StoreProduct.kt

```diff
@@ -0,0 +1,28 @@
+package com.revenuecat.purchases.models
+
+import android.os.Parcelable
+import com.android.billingclient.api.ProductDetails
+import com.android.billingclient.api.SkuDetails
+import com.revenuecat.purchases.ProductType
+import com.revenuecat.purchases.models.ComparableData
+import com.revenuecat.purchases.models.StoreProduct
+import com.revenuecat.purchases.parceler.JSONObjectParceler
+import kotlinx.parcelize.Parcelize
+import kotlinx.parcelize.RawValue
+import kotlinx.parcelize.TypeParceler
+import org.json.JSONObject
+
+@Parcelize
+@TypeParceler<JSONObject, JSONObjectParceler>()
+private data class BC5StoreProduct(
+    override val storeProductId: String,
+    override val type: ProductType,
+    override val price: String,
+    override val priceAmountMicros: Long,
+    override val priceCurrencyCode: String,
```

Comment:
Would it make sense to make a Price class that contains formattedString (price), amountMicros and currencyCode? (this could also be used in the pricing phases)

Reply:
Yup, these API changes it should apply for all flavors and implementations

Reply:
in progress! will let you all know once it's done

Reply:
done